docs/go/how-to-develop-a-worker-program-in-go.md

```diff
@@ -73,9 +73,10 @@ Notice that the Task Queue name is the same as the name provided when the [Workf
 
 The name of the Task Queue that is provided to the Worker must be the same Task Queue name that is provided with the invocation of the Workflow Execution.
 
-import SharedWorkersTaskQueueRegistrationNote from '../reminders/note-workers-task-queue-registration-match.md'
+All Workers listening to the same Task Queue name must be registered to handle the exact same Workflows Types and Activity Types.
```

Comment:
Do you want to place these two sentences (here and in the two other topics in this PR) inside a note element as they were when transcluded?

Reply:
No, I wanted to remove the "noteness".